modules/tinymce/src/core/main/ts/api/WindowManager.ts

```diff
@@ -33,8 +33,13 @@ import { Dialog } from './ui/Ui';
  * });
  */
 
+export interface WindowParams {
+  inline?: 'cursor' | 'toolbar';
+  ariaAttrs?: unknown;
```

Comment:
This type is incorrect. It should be `boolean` as per https://github.com/tinymce/tinymce/blob/7a749464a84a8d6a55035b0ef74f4c1fdd2bdc15/modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts#L29

Reply:
done, thank you for catching this